jenkins/opensearch-dashboards/integ-test-opensearch-dashboards.jenkinsfile

```
@@ -0,0 +1,99 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+    parameters {
+        string(
+            defaultValue: '',
+            name: 'opensearch_version',
+            description: 'OpenSearch version to run test on',
+            trim: true
+        )
+
+        string(
+            defaultValue: '',
+            name: 'opensearch_dashboards_version',
+            description: 'OpenSearch Dashboards version to run test on',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'opensearch_build_id',
+            description: 'The build number of OpenSearch for which tests should be run',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'opensearch_dashboards_build_id',
+            description: 'The build number of OpenSearch Dashboards for which tests should be run',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'architecture',
+            description: 'Architecture of the build of OpenSearch/OpenSearch Dashboards',
+            trim: true
+        )
+        string(
+            defaultValue: '',
+            name: 'platform',
+            description: 'Platform of the build of OpenSearch/OpenSearch Dashboards',
+            trim: true
+        )
+    }
+    stages {
+        stage('integ-test') {
+            agent {
+                docker {
+                    label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+                    image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211028'
+                    alwaysPull true
+                }
+            }
+            steps {
+                script {
+                    basePathOpenSearch = getPublicUrl(
+                        name:'Open Search', 
+                        jobName:'distribution-build-opensearch', 
+                        architecture:architecture, 
+                        platform:platform, 
+                        buildNumber:opensearch_build_id,
+                        version: opensearch_version
+                    )
+
+                    basePathOpenSearchDashboards = getPublicUrl(
+                        name:'Open Search Dashboards', 
+                        jobName:'distribution-build-opensearch-dashboards', 
+                        architecture:architecture, 
+                        platform:platform, 
+                        buildNumber:opensearch_dashboards_build_id,
+                        version: opensearch_dashboards_version
+                    )
+
+                    sh "./test.sh integ-test manifests/${opensearch_dashboards_version}/opensearch-dashboards-${opensearch_dashboards_version}-test.yml -p opensearch=${basePathOpenSearch} opensearch-dashboards=${basePathOpenSearchDashboards} --test-run-id ${env.BUILD_NUMBER}"
```

Lets just pass the test manifest path, this is forward compatible with path changes
```suggestion
                    sh "./test.sh integ-test ${TEST-MANIFEST} -p opensearch=${basePathOpenSearch} opensearch-dashboards=${basePathOpenSearchDashboards} --test-run-id ${env.BUILD_NUMBER}"
```